internal/core/swrenderer/draw_functions.rs

```diff
@@ -283,6 +283,7 @@ fn interpolate_color(
 /// PremultipliedRgbaColor can be constructed from a [`Color`] with
 /// the [`From`] trait. This conversion will pre-multiply the color
 /// components
+#[allow(missing_docs)]
```

Comment:
Looks unrelated ;)

Reply:
I realized the new function didn't have docs, so i added the `warn`  ,  but then it warns about undocumented fields in this struct, which i think is fine.